scripts/run_pylint.py

```diff
@@ -72,7 +72,7 @@
 }
 TEST_RC_REPLACEMENTS = {
     'FORMAT': {
-        'max-module-lines': 1960,
+        'max-module-lines': 2000,
```

Comment:
It seems weird to bump this for a unit test file (bigquery/unit_tests/test_table.py right?).
Maybe we should breakup those tests?